pulp_deb/tests/functional/conftest.py

```diff
@@ -104,6 +105,27 @@ def _deb_package_factory(**kwargs):
     return _deb_package_factory
 
 
+@pytest.fixture(scope="class")
+def deb_release_component_factory(apt_release_component_api, gen_object_with_cleanup):
+    """Fixture that generates deb package with cleanup."""
+
+    def _deb_release_component_factory(component, distribution, repository=None, **kwargs):
+        """Create an APT ReleaseComponent.
+
+        :returns: The created ReleaseComponent.
+        """
+        args = {
+            "component": component,
+            "distribution": distribution,
+        }
+        if repository:
+            args["repository"] = repository
+        release_component_object = DebReleaseComponent(**args)
```

Comment:
```suggestion
        release_component_object = DebReleaseComponent(
            component=component,
            distribution=distribution,
            **kwargs
        )
```

This way we can remove the args and `if repository:` block. We can also remove `repository=None` in this case from the function call and let it be handled by `**kwargs` since the latter is not really used anyway.

Reply:
Done.